src/main/kotlin/mate/academy/figure/Figure.kt

```diff
@@ -0,0 +1,80 @@
+package mate.academy.figure
+
+import mate.academy.model.Color
+import mate.academy.service.AreaOfFigureSupplier
+import mate.academy.service.DrawSupplier
+
+sealed class Figure(val color: Color): DrawSupplier {
+}
+
+class Circle(color: Color,
+             private val radius: Double): Figure(color),
+    AreaOfFigureSupplier,
+    DrawSupplier {

```

Comment:
Figure class already implements DrawSupplier interface. Check if this interface here and in similar cases is obligatory